front/src/components/Table/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
```

Comment:
Nazwa komponentu i pliku powinna być BooksTable i powinno to być gdzieś na w folderze ze stroną z książkami - ten komponent nie jest generyczny. Table miałoby sens, gdybyś mógł tu przekazać tablicę czegokolwiek. A w środku komponentu zaciągasz książki, więc to pasuje tylko i wyłącznie do książek

Reply:
Done.